packages/dataviews/src/dataviews-layouts/grid/style.scss

```diff
@@ -1,8 +1,19 @@
 .dataviews-view-grid {
 	margin-bottom: auto;
+	display: grid;
+	gap: $grid-unit-40;
 	grid-template-rows: max-content;
+	grid-template-columns: repeat(auto-fill, minmax(230px, 1fr));
 	padding: 0 $grid-unit-60 $grid-unit-30;
 	container-type: inline-size;
+	/**
+	 * Breakpoints were adjusted from media queries breakpoints to account for
+	 * the sidebar width. This was done to match the existing styles we had.
+	 */
+	@container (max-width: 480px) {
+		padding-left: $grid-unit-30;
+		padding-right: $grid-unit-30;
+	}
```

Comment:
I haven't changed this (only moved it around in the file) but this padding logic could probably be improved to be fully independent of sidebar width (which is specific to the site editor) and gutenberg media queries breakpoints.